src/rust/tests/write.rs

```diff
@@ -129,6 +129,35 @@ fn json_to_fgb() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn json_to_fgb_geometry_only() -> Result<()> {
+    use std::io::Cursor;
+
+    let mut fgb = FgbWriter::create("countries", GeometryType::Point, |fbb, header_args| {
+        header_args.description = Some(fbb.create_string("Point no properties"));
+        header_args.columns = None;
+    })?;
```

Comment:
Here's some debug output showing the header that the rust writer currently generates for this. Note the `columns: Some([])` even tho I manually set it to None in the args.

```
[src/properties_reader.rs:48] self.header() = Header {
    name: Some(
        "test_point",
    ),
    envelope: Some(
        [
            0.0,
            0.0,
            0.0,
            0.0,
        ],
    ),
    geometry_type: Point,
    hasZ: false,
    hasM: false,
    hasT: false,
    hasTM: false,
    columns: Some(
        [],
    ),
    features_count: 1,
    index_node_size: 16,
    crs: Some(
        Crs {
            org: None,
            code: 4326,
            name: None,
            description: None,
            wkt: None,
            code_string: None,
        },
    ),
    title: None,
    description: Some(
        "Point no properties",
    ),
    metadata: None,
}
```